### DISCUSSION TITLE
2006/10/19/%E4%BA%BA%E7%9A%84%E8%84%91%E8%A2%8B%E8%BF%98%E6%98%AF%E5%BE%88%E7%BB%93%E5%AE%9E%E7%9A%84

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#123] [Announcements]
# 2006/10/19/%E4%BA%BA%E7%9A%84%E8%84%91%E8%A2%8B%E8%BF%98%E6%98%AF%E5%BE%88%E7%BB%93%E5%AE%9E%E7%9A%84

昨晚和同事一起打羽毛球，是双打。我同伙拼劲全力一记扣杀，结果直接把球拍拍到了我的前额上。他的球拍当场断裂，而我的头也出了点血。

https://blog.qizhen.xyz/2006/10/19/%E4%BA%BA%E7%9A%84%E8%84%91%E8%A2%8B%E8%BF%98%E6%98%AF%E5%BE%88%E7%BB%93%E5%AE%9E%E7%9A%84

**Comment by ruanqizhen** [comment]
之前博客下的留言：


Yan
2006年10月23日 19:33 编辑
 
      有谋杀的嫌疑。。。又得罪人了吧!
 

回复
wings
2006年10月21日 19:52 编辑
不知道你现在还好么，呵呵，以前还骂过你，希望你能快点好，这一次是真心的祝福你呢，以后要当心~~！

回复
漠漠
2006年10月21日 16:40 编辑
呵呵，生活丰富哈

回复
人淡如菊
2006年10月20日 09:11 编辑
可怜的球拍

回复
Michael
2006年10月20日 08:03 编辑
早安~多多保重身體喔!

回复
yoyo
2006年10月19日 23:09 编辑
那拍子啥牌子阿 什么材料 攻击型还是防守型阿 你的头也硬了点瓦
顶个大包上班回头率肯定高的

回复
Jing
2006年10月19日 22:41 编辑
可怜的脑袋，可怜的球拍。

回复
Ling
2006年10月19日 21:26 编辑
我的高额头就是被不断砸出来的，呜呜呜，起了包也不退下去，然后额头就越来越大了，说好听点高额头显得聪明些，其实，其实，个中痛楚只有自己知道啊。。。

回复
奕瑋
2006年10月19日 11:44 编辑
八成是你站的位置比较碍事。
都这把年纪了，加上那个眼神儿，
不如逮逮小鸡、拔拔羽毛、做几个球来的安全，
也算是锻炼身体呀！
……头回做沙发，哈哈！

回复
